src/types/param.type.ts

```diff
@@ -0,0 +1,3 @@
+export type ParamType = {
```

Comment:
You can remove the Type keyword just for consistency, As I see you're not using the Type keyword for another Types.